sparrow-py/pysrc/sparrow_py/_expr.py

```diff
@@ -11,14 +11,16 @@
 
 import pandas as pd
 import pyarrow as pa
-import sparrow_py
+import sparrow_py as kt
```

Comment:
why `kt`?

Reply:
Leaning towards renaming this to `timelines` and thought `t` was too short, so went with `kt` for "kaskada timelines". We can adjust, but `kt` is also pretty easy to search for compared to `t` ;)